### ISSUE TITLE
[UX] Admin bar: "Manage fields" and "Manage display" items for user accounts unnecessarily nested under an extra "Manage fields" item.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#2448]
This is related/similar to #2447 

![backdrop-issue2448](https://cloud.githubusercontent.com/assets/2423362/21584198/c5221f20-d0f2-11e6-8414-2b8d2097ca11.png)

It should be:

Configuration -> User accounts -> Manage fields
Configuration -> User accounts -> Manage displays

...while it currently is:

Configuration -> User accounts -> Manage fields -> Manage fields
Configuration -> User accounts -> Manage fields -> Manage displays

**Comment by klonos** [comment]
...also, perhaps all the other items under Configuration -> User accounts should be defined as `MENU_LOCAL_TASK` so that they all show as tabs under the "User accounts" page.

**Comment by serundeputy** [comment]
Sandbox:
Website: http://1718.backdrop.backdrop.qa.backdropcms.org
Username: admin
Password: 7roCVPPz

PR: https://github.com/backdrop/backdrop/pull/1718

**Comment by Dinornis** [comment]
Looks good 👍

**Comment by olafgrabienski** [comment]
@Dinornis - I removed the "fixed" status which as far as I know should be added only when a fix is also committed to the Backdrop code. Instead of "fixed" I added "pr - reviewed & tested by the community".

**Comment by serundeputy** [comment]
Thanks @Dinornis and @olafgrabienski we have `RTBC` and we know it works, passes human and automated tests.

Anyone @jenlampton, @klonos, @quicksketch  want to way in if it is a good UX change?

**Comment by quicksketch** [comment]
Thanks @serundeputy for the PR. This is an interesting update to the menu structure. I like the removing of nesting in admin menu, as that's how most users will arrive at this page, rather than drilling down through the admin pages.

So overall, I like the change, though I am ambivalent about making `admin/config/people` the "Manage fields" page. I see it's necessary for this approach to work, so although I prefer the listing version of that page, it has to be the default tab to eliminate the nesting.

This page is the only problem I see in the PR, which currently does not have any active tab when visiting `admin/config/people` directly:

![screenshot from 2017-01-08 21-13-40](https://cloud.githubusercontent.com/assets/100206/21757704/c63ba526-d5e7-11e6-95e7-1554acb96f44.png)

No tab is selected. It should be "Manage fields", indicating the page you are currently on.

**Comment by jenlampton** [comment]
Hi All!  Thanks for the work on this one :)  Overall, I think it's a [UX] improvement to be rid of the extra level of navigation, though I'm in agreement with @quicksketch that I'm not sure how I feel about the "most important" user configuration being the field UI. Not only is this not a great UI to throw at people, but In my experience fields are only added to user accounts on a minority of sites. Would it be possible to move `Manage Fields` and `Manage Display` to the far right of the tabs, and have the default local task be `Account settings`?

**Comment by Dinornis** [comment]
👍
> Would it be possible to move Manage Fields and Manage Display to the far right of the tabs, and have the default local task be Account settings?

**Comment by serundeputy** [comment]
With a significant amount of work left to do and release tomorrow moving this to the `1.6.1` milestone.

**Comment by klonos** [comment]
This just needs one of the `MENU_LOCAL_TASK`s to be defined as `MENU_DEFAULT_LOCAL_TASK` + a bit of weight adjustment for all items. I guess we all seem to agree that the default needs to be the "Account settings" menu. As for menu order, I suggest keeping the order prior to this change as much as possible, so I think that @jenlampton's suggestion makes the most sense:

>Would it be possible to move `Manage Fields` and `Manage Display` to the far right of the tabs, and have the default local task be `Account settings`?

In other words, this:

**Account settings**
Permissions
Roles
Manage fields
Manage display
Account emails

**Comment by klonos** [comment]
@serundeputy thanx for taking a stab at this. I just filed a PR over your branch that finishes this off. ...well kinda because now we have lost the the "User accounts" section from the listing under `/admin/config` and I can't figure out how to get it back.

**Comment by klonos** [comment]
I think that this is a great UX improvement. Previously, in order to navigate to a "sibling" menu of "Account settings", one had to either make multiple hovers over the admin bar or multiple clicks (on mobile where hover is not possible). Now with the sibling menu items as tabs, things are only a click away.

I think we should implement this pattern in most (if not all) the landing pages under `/admin/config/xyz` (and elsewhere perhaps?) where we replace the landing pages with the lists of links with `MENU_DEFAULT_LOCAL_TASK`s and add the rest of the siblings as `MENU_LOCAL_TASK`s. This makes perfect sense especially for menus that have a single child item like Configuration -> Administration. I mean, what's the point of this page besides requiring another click?:

![backdrop-admin_bar_landing_page-single_item](https://cloud.githubusercontent.com/assets/2423362/21965913/5a76c6c8-dbbd-11e6-9129-94ddf174f425.png)

**Comment by klonos** [comment]
...same goes for Configuration -> Content authoring

**Comment by Al-Rozhkov** [comment]
PR: https://github.com/backdrop/backdrop/pull/1814

Pull request should fix initial issue. But I don't know how to make items under Configuration -> User accounts work as tabs and keep them on Configuration page (/admin/config) at the same time.

Maybe it is better to create another issue "Configuration items as tabs" for consistent UX.

**Comment by klonos** [comment]
Not working on the PR sandbox @Al-Rozhkov:

![image](https://cloud.githubusercontent.com/assets/2423362/24443316/e5deea6e-14ad-11e7-9168-856f3ac08dc3.png)

**Comment by Al-Rozhkov** [comment]
Yes. Sorry, I'm newbie at making PR process. My PR is closed now.

**Comment by klonos** [comment]
@Al-Rozhkov no need to apologize. I'm a newbie myself 😄

**Comment by docwilmot** [comment]
I cant find the latest PR for this. [This comment](https://github.com/backdrop/backdrop-issues/issues/2448#issuecomment-272720328) suggests @klonos did a new one?

**Comment by BWPanda** [comment]
> perhaps all the other items under Configuration -> User accounts should be defined as MENU_LOCAL_TASK so that they all show as tabs under the "User accounts" page.

> Anyone [...] want to way in if it is a good UX change?

**I don't like this idea...** It looks good in the screenshots on a plain Backdrop site, but what happens when contrib modules add pages under this sub-menu? Will they all get tabs on this new page too, or will only some (core) pages get tabs and other (contrib) pages get regular menu items? How will that look/work? Here are some examples of modules that add pages to the User Accounts menu item:

- https://github.com/backdrop-contrib/hybridauth/blob/1.x-2.x/hybridauth.info
- https://github.com/backdrop-contrib/ip_blocking/blob/master/ip_blocking.info
- https://github.com/backdrop-contrib/login_allowlist/blob/main/login_allowlist.info
- https://github.com/backdrop-contrib/ip2country/blob/1.x-1.x/ip2country.info

As @klonos suggested above:

> I think that we should implement this pattern in most (if not all) the landing pages under /admin/config/xyz (and elsewhere perhaps?)

If we do this in one place, we should do it everywhere (for consistent UX). But I don't believe it'll work when any number of contrib modules can add new pages to any menu item.

So I suggest just sticking with the original request to move the two User Account field menu items out of the sub-menu. Here's a PR to that effect: https://github.com/backdrop/backdrop/pull/3478

It seems that this was always the intention with these pages, there was just a typo in the function name!

**Comment by BWPanda** [comment]
![image](https://user-images.githubusercontent.com/2385329/103528646-69364980-4ed8-11eb-901e-4951b4b0a583.png)
